### ISSUE TITLE
Basic example does not work

### ISSUE THREAD

**Issue by kenden** [merll/docker-fabric#1]
I am following the documentation at:
https://docker-fabric.readthedocs.org/en/latest/installation.html

pip install docker-fabric
sudo apt-get install lxd-docker-1.5.0 (on host and remote machine)
sudo apt-get install socat (on host and remote machine)
usermod -aG docker jenkins
echo "from dockerfabric import tasks as docker" > fabfile.py

But I get the error: 

```
$ fab docker.version
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/fabric/main.py", line 743, in main
    *args, **kwargs
  File "/usr/local/lib/python2.7/dist-packages/fabric/tasks.py", line 424, in execute
    results['<local-only>'] = task.run(*args, **new_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/fabric/tasks.py", line 174, in run
    return self.wrapped(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/dockerfabric/tasks.py", line 150, in version
    output = docker_fabric().version()
  File "/usr/local/lib/python2.7/dist-packages/dockerfabric/apiclient.py", line 45, in get_connection
    return self.get(key, DockerFabricClient, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/dockerfabric/base.py", line 15, in get
    e = d(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/dockerfabric/apiclient.py", line 94, in __init__
    local_port = get_local_port(init_local_port)
  File "/usr/local/lib/python2.7/dist-packages/dockerfabric/base.py", line 24, in get_local_port
    return int(init_port) + current_offset
TypeError: int() argument must be a string or a number, not 'NoneType'
```

I get the same with:

```
$ fab -H "localhost" docker.version
$ fab -H "10.0.101.133" docker.version
```

Note that SSH works:

```
$ ssh 10.0.101.133
Welcome to Ubuntu 14.04.2 LTS (GNU/Linux 3.13.0-46-generic x86_64)
jenkins@test:~$
```

That's on Ubuntu 14.10

I have the following version installed:
docker-fabric (0.3.1)
docker-map (0.2.2)
docker-py (1.0.0)

**Comment by merll** [comment]
Thank you for the feedback. This problem applied to direct TCP connections (e.g. to localhost) where no local port is assigned. The bug is now fixed in 0.3.2.

**Comment by kenden** [comment]
Thanks!
